heron/api/src/scala/com/twitter/heron/streamlet/scala/converter/ScalaToJavaConverter.scala

```diff
@@ -27,6 +32,17 @@ object ScalaToJavaConverter {
       override def get(): T = f()
     }
 
+
+  def toJavaSource[T](source: Source[T]): com.twitter.heron.streamlet.Source[T] = {
+    new com.twitter.heron.streamlet.Source[T] {
+      override def setup(context: Context): Unit = source.setup(context)
+
+      override def get(): Collection[T] = scala.collection.JavaConverters.asJavaCollectionConverter(source.get).asJavaCollection
```

Comment:
"scala.collection.JavaConverters." is not necessary.

Reply:
It is actually, I noticed that I was getting compilation errors if I didnt include this even though I had the appropriate import statement

Reply:
@skanjila - any idea why the compilation errors occurred?

Reply:
I will dig into that issue on the next commit

Reply:
hmm. interesting. another note is that "asJavaCollectionConverter(...).asJavaCollection" looks a bit strange. from function name, it seems asJavaCollectionConverter(...) should return a Java collection, hence the following asJavaCollection looks a bit confusing. Any idea why this is needed?

Reply:
I believe this just takes an input collection and converts it to a Java collection, again I will dig into this as well on the next commit to simplify this, here's the issue I created to track the import alias issue and this: https://github.com/twitter/heron/issues/2772